src/Composer/Util/Http/CurlDownloader.php

```diff
@@ -305,7 +305,9 @@ public function abortRequest(int $id): void
         if (isset($this->jobs[$id], $this->jobs[$id]['curlHandle'])) {
             $job = $this->jobs[$id];
             curl_multi_remove_handle($this->multiHandle, $job['curlHandle']);
-            curl_close($job['curlHandle']);
+            if (PHP_VERSION_ID < 80000) {
```

Comment:
```suggestion
            if (\PHP_VERSION_ID < 80000) {
```

Using a qualified constant there allows OPCache to resolve the condition at compile time, eliminating it entirely.